SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -314,4 +314,9 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	  * the map's config with a 1x1 grid.
 	  */
 	TOptional<bool> bOverrideMultiWorker;
+
+	/**
+	  * This will enable warning messages for ActorSpawning that could be legitimate but is likely to be an error.
```

Comment:
This is slightly more than warning messages, it stops the actor deletion - should we distinguish? ie. just make this flag prevent the warning message when dynamically spawning actors before beginplay/isready is set?

Reply:
I really wavered on the name of the flag, so I'm totally happy to change that. 

Independent of that - the long term solution is to move the tear off into the LBStrategy, at which point the flag would only be about the debugging messages. So I went for long term use rather than being super correct about immediate behaviour. I'm happy to change that if you think I should.

Reply:
For clarity - in the long term solution, checking the flag also happens in the LBStrategy, not here in the NetDriver.